tests/integration/orchestration/prefect/flows/test_salesforce.py

```diff
@@ -0,0 +1,17 @@
+"""'test_salesforce.py'."""
+
+from viadot.orchestration.prefect.flows import salesforce_to_adls
+
+
+def test_salesforce_to_adls():
+    """Test SalesForce prefect flow."""
+    state = salesforce_to_adls(
+        azure_key_vault_secret="salesforce-prod",
+        env="dev",
+        table="Contact",
+        adls_path="raw/dyvenia_sandbox/salesforce/salesforce.csv",
+        adls_azure_key_vault_secret="app-azure-cr-datalakegen2-dev",
```

Comment:
I think secret names are sensitive and should be taken from the config. Probably same for ADLS path.

Reply:
You are probably right. But as I mentioned in other feedback, this is a transcription from the original, so I would recommend to add this to a backlog task.

Reply:
On this one I can't agree, since it's sensitive info, I don't think it should go into the lib at all.

Reply:
Changes applied on [created integration test file.](https://github.com/dyvenia/viadot/pull/1036/commits/065f5fbb22b4d99efecd7b1d1918f40a5ccd1587) and [e7a0ef0](https://github.com/dyvenia/viadot/pull/1036/commits/e7a0ef0df4688da95ab8c52783186afac2de1bd1)